python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_wingweight.py

```diff
@@ -0,0 +1,462 @@
+
+# %%
+squared_src_indices = corr_analysis.computeSquaredSRC(True)
+print(squared_src_indices)
+
+# %%
+# And its associated graph:
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(
+    squared_src_indices, inputNames, 'Squared SRC coefficients - Wing weight')
+view = otv.View(graph)
+
+# %%
+#
+
+# %%
+# SRRC coefficients
+# --------------------
+# We compute here `SRRC` coefficients using the `CorrelationAnalysis`.
+
+# %%
+srrc_indices = corr_analysis.computeSRRC()
+print(srrc_indices)
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(
+    srrc_indices, inputNames, 'SRRC coefficients - Wing weight')
+view = otv.View(graph)
+
+# %%
+# Pearson coefficients
+# -----------------------
+# We compute here the Pearson :math:`\rho` coefficients using the `CorrelationAnalysis`.
+
+# %%
+pearson_correlation = corr_analysis.computePearsonCorrelation()
+print(pearson_correlation)
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(pearson_correlation,
+                                                             inputNames,
+                                                             "Pearson correlation coefficients - Wing weight")
+view = otv.View(graph)
+
+# %%
+# Spearman coefficients
+# -----------------------
+# We compute here the Spearman :math:`\rho_s` coefficients using the `CorrelationAnalysis`.
+
+# %%
+spearman_correlation = corr_analysis.computeSpearmanCorrelation()
+print(spearman_correlation)
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(spearman_correlation,
+                                                             inputNames,
+                                                             "Spearman correlation coefficients - Wing weight")
+view = otv.View(graph)
+plt.show()
+
+# %%
+#
+# The different computed correlation estimators show that the variables :math:`S_w, A, N_z, t_c` seem to be the most correlated with the wing weight.
+# Pearson and Spearman coefficients do not reveal any linear nor monotonic correlation as no coefficients are equal to +/- 1.
+# Coefficients about :math:`t_c` are negative revealing a negative correlation with the wing weight, that is consistent with the model expression.
+
+
+# %%
+# Taylor expansion importance factors
+# -----------------------------------
+# We compute here the Taylor expansion importance factors using `TaylorExpansionMoments`.
+
+# %%
+
+# %%
+# We create a distribution-based RandomVector.
+X = ot.RandomVector(m.distributionX)
+
+# %%
+# We create a composite RandomVector Y from X and m.model.
+Y = ot.CompositeRandomVector(m.model, X)
+
+# %%
+# We create a Taylor expansion method to approximate moments.
+taylor = ot.TaylorExpansionMoments(Y)
+
+# %%
+# We get the importance factors.
+print(taylor.getImportanceFactors())
+
+# %%
+# We draw the importance factors
+graph = taylor.drawImportanceFactors()
+graph.setTitle('Taylor expansion imporfance factors - Wing weight')
+view = otv.View(graph)

```

Comment:
This is a limitation of the service, but here 2 legends are superposed:

![image](https://user-images.githubusercontent.com/31989332/183741914-2c62499c-4bd0-4d74-8aac-80f2075212a0.png)

@jschueller @mbaudin47 Maybe we could define a threshold which would remove shares with a too small percentage?